main/main.cpp

```diff
@@ -44,9 +50,43 @@ static void initResources()
 
 int main(int argc, char** argv)
       {
+      // Force the 8-bit text encoding to UTF-8. This is needed on Windows, which would otherwise default to the local ANSI code page.
+      QTextCodec::setCodecForLocale(QTextCodec::codecForName("UTF-8"));
+
       initResources();
 
       ModulesSetup::instance()->setup();
 
-      return Ms::runApplication(argc, argv);
+#ifdef Q_OS_WIN
```

Comment:
I believe this is only needed for MSVC, not for MinGW, but I don't know whether it harms there.

Reply:
`#if (defined (_MSCVER) || defined (_MSC_VER))`

Reply:
There are no Qt-specific platform checks available specifically for MinGW, so I made sure that the changes would work under both MSVC and MinGW. If you are able to test this with MinGW to confirm, please do.

Reply:
It does for MSVC. And for Windows + GCC == MinGW

Reply:
> `#if (defined (_MSCVER) || defined (_MSC_VER))`

Would you like me to change it to that?

Reply:
Unless we can prove that this change works with MinGW too, yes, please

Reply:
Done.